<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QWidget" name="RegistrationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login/Registartion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(99, 69, 44, 250), stop:0.422886 rgba(175, 131, 95, 250), stop:0.517413 rgba(180, 125, 80, 250), stop:1 rgba(99, 69, 44, 250));</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>139</x>
         <y>89</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:none;</string>
       </property>
       <property name="text">
        <string>Choose your role: </string>
       </property>
      </widget>
      <widget class="QPushButton" name="teacherBtn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>250</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>Teacher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pupilBtn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>270</y>
         <width>250</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>Pupil</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="confirmLoginBtnP">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>380</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yrsa</family>
         <pointsize>16</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(111, 79, 52);
border-color: rgb(233, 191, 159);
color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>CONFIRM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>272</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>321</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your password:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QPushButton" name="teacherRegistBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>140</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>Registration</string>
       </property>
      </widget>
      <widget class="QPushButton" name="teacherLoginBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>230</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QPushButton" name="pupilRegistBtn_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>140</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>Registration</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pupilLoginBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>230</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_7">
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>321</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>270</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="registrBtnT">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>380</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yrsa</family>
         <pointsize>16</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(111, 79, 52);
border-color: rgb(233, 191, 159);
color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>CONFIRM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>321</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your password:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QPushButton" name="confirmLoginBtn">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>370</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yrsa</family>
         <pointsize>16</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(111, 79, 52);
border-color: rgb(233, 191, 159);
color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>CONFIRM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>241</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>321</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>270</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>321</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your password:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="registrBtnP">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>370</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yrsa</family>
         <pointsize>16</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(111, 79, 52);
border-color: rgb(233, 191, 159);
color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string>CONFIRM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>321</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter your name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>346</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 191, 159);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>321</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string>Enter class` code:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
